### PR TITLE
Add !guildinfo command which shows information on the current or any guild

### PR DESCRIPTION
**Pull request by Iapetus-11**
image

### CODE REVIEW

File: ClemBot.Bot/bot/cogs/guild_info_cog.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import bot.extensions as ext
+from bot.consts import Colors
+
+class GuildInfoCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @ext.command()
+    @ext.long_help('Shows information on the current guild/Discord server.')
```
Comment (Jay-Madden) [inline ClemBot.Bot/bot/cogs/guild_info_cog.py:14]:
```suggestion
    @ext.long_help('Shows information on the current Guild/Discord server.')
```

File: ClemBot.Bot/bot/cogs/guild_info_cog.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        bot_count = len([m for m in guild.members if m.bot])
+        channel_count = len(guild.text_channels) + len(guild.voice_channels)  # excludes categories
+        formatted_roles = ' '.join(reversed([r.mention for r in guild.roles if r.id != guild.id]))
+        age = arrow.get(guild.created_at)
+        display_age = f"{age.format('MMM D, YYYY')}, {age.humanize()}"
+
+        # try:
+        #     ban_count = len(await asyncio.wait_for(guild.bans(), 1))
+        # except asyncio.TimeoutError:  # so many bans, eeee
+        #     ban_count = 'unknown'
```
Comment (Jay-Madden) [inline ClemBot.Bot/bot/cogs/guild_info_cog.py:33]: you left 
```suggestion

```

File: ClemBot.Bot/bot/cogs/guild_info_cog.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+        base = '» **{}:** {}'
+
+        embed.description = '\n'.join([
+            base.format('Created At', display_age),
+            base.format('Owner', guild.owner.mention),
+            base.format('Member Count', member_count),
+            base.format('Bot Count', bot_count),
+            # base.format('Ban Count', ban_count),
```
Comment (Jay-Madden) [inline ClemBot.Bot/bot/cogs/guild_info_cog.py:42]:
```suggestion
```

File: ClemBot.Bot/bot/cogs/guild_info_cog.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            base.format('Channel Count', channel_count),
+            base.format('Roles', formatted_roles),
+        ])
+
+        embed.set_footer(text=str(ctx.author), icon_url=ctx.author.avatar_url)
```
Comment (Jay-Madden) [inline ClemBot.Bot/bot/cogs/guild_info_cog.py:47]: this needs to be `author.display_avatar.url`

**Review by Jay-Madden — APPROVED** [review]

**Review by Jay-Madden — CHANGES_REQUESTED** [review]
change request cuz i fucked it up the first time

File: ClemBot.Bot/bot/cogs/guild_info_cog.py (hunk 4 above)
Comment (Iapetus-11) [reply ClemBot.Bot/bot/cogs/guild_info_cog.py:47]: But it works as is because it's str()-ified internally

Comment (Jay-Madden) [reply ClemBot.Bot/bot/cogs/guild_info_cog.py:42]: :+1:

File: ClemBot.Bot/bot/cogs/guild_info_cog.py (hunk 1 above)
Comment (Iapetus-11) [reply ClemBot.Bot/bot/cogs/guild_info_cog.py:14]: Guild isn't capitalized by itself in the Discord API docs unless it's a title or at the start of a sentence.

File: ClemBot.Bot/bot/cogs/guild_info_cog.py (hunk 4 above)
Comment (Jay-Madden) [reply ClemBot.Bot/bot/cogs/guild_info_cog.py:42]: No, avatar_url doesnt exist in d.py 2.0

Comment (Iapetus-11) [reply ClemBot.Bot/bot/cogs/guild_info_cog.py:42]: ah ok

**Review by Jay-Madden — APPROVED** [review]
lgtm

**Pull request merged**